lib/matplotlib/tests/test_transforms.py

```diff
@@ -987,12 +987,6 @@ def test_transformed_path():
                     [(0, 0), (r2, r2), (0, 2 * r2), (-r2, r2)],
                     atol=1e-15)
 
-    # Changing the path does not change the result (it's cached).
-    path.points = [(0, 0)] * 4
-    assert_allclose(trans_path.get_fully_transformed_path().vertices,
```

Comment:
Dropping this test worries me.

On one hand, it is surprising to me why keep a reference to the transform but copy the path, but if this test is here it suggests that _something_ is depending on this behavior and we sort out what it was or be sure that nothing depends on this.

I have been bit too many times by "I don't see why we need that..." and then a user reports to us why ! 

https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence

Reply:
Well the problem as far as I can tell is that this test wasn't doing anything - `.points` is not a path attribute and not referenced anywhere else. When I update it to `path._vertices` off of main, it does fail. I'm happy to add it back in with the original code and open a ticket to investigate it if you think that's the best path.